react/features/invite/functions.js

```diff
@@ -825,13 +825,15 @@ export function inviteSipEndpoints( // eslint-disable-line max-params
         return Promise.resolve();
     }
 
+    const baseUrl = window.location.href.toLowerCase().replace(`/${roomName}`, '');
```

Comment:
This (well, and previous form neither) won't work on mobile. You can extract the URL from base/connection instead.